### PR TITLE
[Docs] update: tutorials ja | AUTOPIPELINE.md

### PR DESCRIPTION
**Pull request by YasunaCoffee**
# What does this PR do?

**Changes included in this pull request**

Related to #6309

What is this PR about ?

I translated the following files to implified Japanese:

- _toctree.yml
- docs/ja/tutorials/autopipeline.md

here is the web page which is translated
https://huggingface.co/docs/diffusers/tutorials/autopipeline

What should you check ?

Grammar and orthograph
If my pull requests follows GitHub/open source contribution good practices (I am a beginner)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
[Error with Doc-Builder: Error 404 on Section Pages in Doc-Builder Preview
](https://discuss.huggingface.co/t/error-with-doc-builder-error-404-on-section-pages-in-doc-builder-preview/68379/1)
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
- Docs: 🤗 @stevhliu
- Docs: Japanese-Speaker @Prgckwb, @ratovia, @morieeeenyo, @m-qme

### CODE REVIEW

File: docs/source/ja/tutorials/autopipeline.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import torch
+
+pipeline_text2img = AutoPipelineForText2Image.from_pretrained(
+    "runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16, use_safetensors=True
+)
+print(type(pipeline_text2img))
+"<class 'diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline'>"
+```
+
+そして、[`from_pipe()`] (https://huggingface.co/docs/diffusers/v0.25.1/en/api/pipelines/auto_pipeline#diffusers.AutoPipelineForImage2Image.from_pipe)は、もとの`"stable-diffusion"` パイプラインのクラスである [`StableDiffusionImg2ImgPipeline`] にマップします:
```
Comment (stevhliu) [inline docs/source/ja/tutorials/autopipeline.md:137]:
```suggestion
そして、[from_pipe()](https://huggingface.co/docs/diffusers/v0.25.1/en/api/pipelines/auto_pipeline#diffusers.AutoPipelineForImage2Image.from_pipe)は、もとの`"stable-diffusion"` パイプラインのクラスである [`StableDiffusionImg2ImgPipeline`] にマップします:
```

**Review by stevhliu — COMMENTED** [review]
Thanks for the translation, LGTM! I'll allow the other folks tagged some time to review the content before merging :)

**Review by Prgckwb — COMMENTED** [review]
LGTM 🤗

**Comment by YasunaCoffee** [comment]
Hi, @stevhliu 

This pull request review is complete!
Please merge it 🤝

File: docs/source/ja/tutorials/autopipeline.md (hunk 1 above)
Comment (stevhliu) [reply docs/source/ja/tutorials/autopipeline.md:137]: This comment still needs to be addressed otherwise the link appears incorrectly.

Comment (YasunaCoffee) [reply docs/source/ja/tutorials/autopipeline.md]: Thanks! I fixed this issue.

**Comment by YasunaCoffee** [comment]
Hi, @stevhliu 🤗
I finished to fix your FB, thanks!

**Review by stevhliu — APPROVED** [review]
Great work! 👍

**Pull request merged**